camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -335,11 +336,11 @@ import {
 import { Component, Mixins, Prop } from "vue-property-decorator";
 import { CustomEventPayload, TaskPayload, UserSearchListLabelPayload } from "../models/TaskPayload";
 import {
-  TASK_FILTER_LIST_DEFAULT_PARAM,
+  TaskListSortType,
   findFilterKeyOfAllTask,
   getTaskFromList,
-  getUserName,
-} from "../services/utils";
+  getUserName
+} from "../services";
```

Comment:
Can you import all the `services` and `models` this way in this file?

Reply:
There are multiple import which are importing `../services/camunda-rest.ts`, `../services/formio-token.ts`. Instead of having multiple imports from multiple files. With the latest change:

Can you import everthing `../services` and `../models`?